src/migration/migrationRules.js

```diff
@@ -1463,5 +1463,44 @@ module.exports = {
     logger.info('Removing applications / dataSources from north / south settings.')
     config.north = config.north.applications
     config.south = config.south.dataSources
+
+    await createFolder('./logs')
+    logger.info('Removing file logs file name from config. Default will be "logs/journal.log".')
+    const oldLogFileName = path.basename(config.engine.logParameters.fileLog.fileName)
+    const newFileLogFilePath = path.resolve('./logs', 'journal.log')
+    try {
+      await fs.rename(oldLogFileName, newFileLogFilePath)
+    } catch (err) {
+      if (err.code !== 'ENOENT') {
+        logger.error(err)
+      }
+    }
+    delete config.engine.logParameters.fileLog.fileName
+
+    const oldLogSqlFileName = path.basename(config.engine.logParameters.sqliteLog.fileName)
+    const newFileLogSqlPath = path.resolve('./logs', 'journal.db')
+    try {
+      await fs.rename(oldLogSqlFileName, newFileLogSqlPath)
+    } catch (err) {
+      if (err.code !== 'ENOENT') {
+        logger.error(err)
+      }
+    }
+    delete config.engine.logParameters.sqliteLog.fileName
+
+    try {
+      await fs.rm(path.resolve('./migration-journal.log'), { force: true })
```

Comment:
Is it ok to just delete the old migration log?

Reply:
My point is that both these files (migration-journal.log and OIBus-main-journal.log) are used for debugging purposes when OIBus fails to (re)start. I think it's safe to remove them now.

@jfhenon what is your opinion on that ?